From 803375901e15313b668a26f3643436636b9b026c Mon Sep 17 00:00:00 2001
From: Jordan Salm <jordans@validmanufacturing.com>
Date: Thu, 23 Jan 2020 14:51:46 -0800
Subject: [PATCH 07/10] Add standard /usr/lib path to rpath

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 782d59f075..c750bbc5bf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -428,6 +428,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "QNX")
 endif()
 
 SET(RPATH_DIRECTORIES
+  ${CMAKE_INSTALL_LIBDIR}
   ${CMAKE_INSTALL_PREFIX}/bin/plugins
   /usr/local/lib
   /usr/local
-- 
2.24.1

