From d3c65d391adac28f1265281ff663dcbd0bf51a5a Mon Sep 17 00:00:00 2001
From: Phong Tran <tranmanphong@gmail.com>
Date: Sun, 12 Feb 2017 18:01:54 +0700
Subject: [PATCH 10/10] Add find path of gstconfig in yocto env

---
 cmake/Modules/FindGstreamer-1.0.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/Modules/FindGstreamer-1.0.cmake b/cmake/Modules/FindGstreamer-1.0.cmake
index 8f1d69d01..c123a134d 100644
--- a/cmake/Modules/FindGstreamer-1.0.cmake
+++ b/cmake/Modules/FindGstreamer-1.0.cmake
@@ -7,7 +7,7 @@ FIND_PATH(GSTREAMER_gst_INCLUDE_DIR gst/gst.h
                                     ENV INCLUDE DOC "Directory containing gst/gst.h include file")
 
 FIND_PATH(GSTREAMER_gstconfig_INCLUDE_DIR gst/gstconfig.h
-                                          PATHS ${GSTREAMER_DIR}/include ${GSTREAMER_DIR}/lib/include /usr/local/include/gstreamer-1.0 /usr/include/gstreamer-1.0 /usr/local/lib/include/gstreamer-1.0 /usr/lib/include/gstreamer-1.0
+                                          PATHS ${GSTREAMER_DIR}/usr/lib/gstreamer-1.0/include ${GSTREAMER_DIR}/include ${GSTREAMER_DIR}/lib/include /usr/local/include/gstreamer-1.0 /usr/include/gstreamer-1.0 /usr/local/lib/include/gstreamer-1.0 /usr/lib/include/gstreamer-1.0
                                           ENV INCLUDE DOC "Directory containing gst/gstconfig.h include file")
 
 FIND_LIBRARY(GSTREAMER_gstaudio_LIBRARY NAMES gstaudio-1.0 libgstaudio-1.0
-- 
2.11.0

